src/main/resources/db/migration/V4__Create_Timesheet.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE timesheet (
+  id BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,
+  project_id BIGINT NOT NULL,
+  employee_id BIGINT NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  description VARCHAR(255),
+  allocation DOUBLE PRECISION NOT NULL,
+  created TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+  updated TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+  CONSTRAINT pk_timesheet PRIMARY KEY (id)
+);
+
+ALTER TABLE timesheet ADD CONSTRAINT FK_TIMESHEET_ON_EMPLOYEE FOREIGN KEY (employee_id) REFERENCES employee (id);
+
+ALTER TABLE timesheet ADD CONSTRAINT FK_TIMESHEET_ON_PROJECT FOREIGN KEY (project_id) REFERENCES project (id);
+
+INSERT INTO timesheet(project_id,employee_id,name,description,allocation,created,updated)
+VALUES
+    (1,1,'test','testing',100,CURRENT_TIMESTAMP, CURRENT_TIMESTAMP ),
+    (1,1,'test2','testing delete',100,CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)
```

Comment:
Not sure if it's needed but i'd add semicolon after the last sql statement too just in case.